dexs/reya/index.ts

```diff
@@ -0,0 +1,57 @@
+import { FetchOptions, FetchResult, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import ADDRESSES from '../../helpers/coreAssets.json';
+
+const eventAbis = {
+  event_order:
+    "event PassivePerpMatchOrder(uint128 indexed marketId, uint128 indexed accountId, int256 orderBase, (uint256 protocolFeeCredit, uint256 exchangeFeeCredit, uint256 takerFeeDebit, int256[] makerPayments, uint256 referrerFeeCredit) matchOrderFees, uint256 executedOrderPrice, uint128 referrerAccountId, uint256 blockTimestamp)",
+};
+
+const CONFIG = {
+  priceDecimals: 18,
+  baseDecimals: 18,
+  quoteDecimals: 6
+}
+
+const fetch = async (options: FetchOptions): Promise<FetchResult> => {
+  const dailyVolume = options.createBalances();
+  const dailyFees = options.createBalances();
+  const dailyRevenue = options.createBalances()
+
+  const logs = await options.getLogs({
+   target: '0x27e5cb712334e101b3c232eb0be198baaa595f5f',
```

Comment:
noticed that all the logs had this as source, is there is chance that it is some other contract?